src/App.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import './App.css';
+import Counter from './components/counter';
+import SearchForm from './components/searchform';
+import GenreSelect from './components/genreselect';
+
+function App() {
+  
+  const genres = ['All', 'Action', 'Romance', 'Horror' , 'Adventure', 'Comedy', 'Drama', 'Sci-Fi'];
+  const [selectedGenre, setSelectedGenre] = useState('All');
+
+  const handleGenreSelect = (genre) => {
+    setSelectedGenre(genre);
+    alert(`Performing a search for selected genre: ${genre}`);
+  };
+
+  const handleSearch = (query) => {
+    // Replace this with your desired search logic
+    alert(`Performing a search for your movie: ${query}`);
+  };
+
+  return (
+    <div className="App">
+      <Counter initialValue={10} />
+      <h1>FIND YOUR MOVIE</h1>
+      <SearchForm onSearch={handleSearch} />
```

Comment:
The component should accept two properties:
0. Initial search query. Use the value to set the initial value for the input.
1. A "onSearch" callback property.

Reply:
Added the changes in this commit: [18eaaa3](https://github.com/Abhinav-apps/netflixroulette/pull/1/commits/18eaaa3a3d6c770d565593fd14df0f4139456ca5)

So now the search form will be prepopulated by an initial search query as shown below.
<img width="960" alt="image" src="https://github.com/Abhinav-apps/netflixroulette/assets/76530299/6c505ed1-d6e6-4efe-b173-7e4cb06b74ca">